coderadar-rest/src/main/java/io/reflectoring/coderadar/rest/analyzerconfig/UpdateAnalyzerConfigurationController.java

```diff
@@ -25,7 +25,7 @@ public UpdateAnalyzerConfigurationController(
   }
 
   @PostMapping(path = "/projects/{projectId}/analyzers/{analyzerConfigurationId}", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity updateAnalyzerConfiguration(
+  public ResponseEntity<Object> updateAnalyzerConfiguration(
```

Comment:
Maybe return `ResponseEntity<HttpStatus>` instead of `ResponseEntity<Object>` and do error handling in the `ExceptionHandlerController` ?

Reply:
I'll work on the ErrorHandler in the next PR